### ISSUE TITLE
[Toolkit feature]: Drag and resize UI inwards, or add arrow to close all (to get full horizontal viewport)

### ISSUE THREAD

**Issue by PappaSlask** [NVIDIAGameWorks/rtx-remix#334]
### What is the context of the feature/improvement?

Would love to have some freedom and flexibility when it comes to the UI and menus. Resizing, maybe re-organizing or moving stuff around. But at least let us make it smaller in width please.

### Please describe the feature/improvement?

UI takes up too much screen estate. Let users customize their workspace a little. Also make it possible to extend properties menus downwards to make them bigger.

### What would be your solution? (optional)

...

### Version

2024.1.1

**Comment by PapaSlask** [comment]
> Hey @PapaSlask ,
> 
> > Also make it possible to extend properties menus downwards to make them bigger.
> 
> Are you talking about this:
> 
>  2024-01-25.09-36-25.mp4 
> ?

Yes exactly! I didn't know you had to press and hold that tiny grey slider to be able to do it! Some sort of visual feedback indicator would be great for this. I think right now it's not obvious how to do it. I suggest having it like Composer, when mouse-over the border line, the line should turn blue.

Also, I prepared a design idea in a picture that I think would be useful. It consists of a gray arrow to close the panel. Right now you have to click on the dark gray surface area to the left to close it, which isn't obvious to a user. I also suggest the ability to resize the whole panel not just to the right to make it bigger, but also to the left to make it smaller, I marked this slider with a red box. The slider does get highlighted, so that's a nice visual feedback, however I think it should be blue as well. White works sure, but blue contrasts better to gray.
![idea](https://github.com/NVIDIAGameWorks/rtx-remix/assets/11363281/ad8430f4-5fa7-400e-bc23-09a9e05f09c3)

**Comment by nvdamien** [comment]
Hi @PappaSlask ,

Yes, there are some nice discussion happening about the layout. I will keep you updated.
Filled an internal ticket to link yours: REMIX-2750

**Comment by PappaSlask** [comment]
> Hi @PappaSlask ,
> 
> Yes, there are some nice discussion happening about the layout. I will keep you updated. Filled an internal ticket to link yours: REMIX-2750

Hello Damien, any update on this? I'm still facing this issue, and the menus are taking up WAY too much real estate. At least give us an arrow to collapse the leftmost menu which contains the Captures / Setup, Asset Replacement and Mod Packaging, since those are not needed 90% of the time when working in Toolkit. Thank you!

**Comment by nvdamien** [comment]
Hey @PappaSlask ,

Last update is: we may have found a technical solution for it. 

Btw, I don't know if you are aware of this but if you click on the left menu, on an empty zone, here:

![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/128181406/cd9c3bd6-001e-4768-b591-89e64fa026df)

it will close the other menu.

**Comment by PappaSlask** [comment]
@nvdamien Thanks a lot for the update, I'm glad to hear you guys cracked this one. And yeah, I'm aware of the trick to click on that gray empty space, however, that will close the one menu that is actually useful to us all the time. It's the leftmost menu that feels like dead weight, make that baddboy collapsible and your my hero! :)
